app/src/main/java/com/mobileproto/hireddit/hireddit/MainFragment.java

```diff
@@ -1,29 +1,108 @@
 package com.mobileproto.hireddit.hireddit;
 
+import android.app.Activity;
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
 
 /**
- * A placeholder fragment containing a simple view.
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link MainFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link MainFragment#newInstance} factory method to
+ * create an instance of this fragment.
  */
 public class MainFragment extends Fragment
 {
-    String TAG = "MainFragment Debug";
-    private View view;
+    String DEBUG_TAG = "MainFragment Debug";
+
+    private OnFragmentInteractionListener mListener;
+
+    @Bind(R.id.alienImage) ImageView alienImage;
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @return A new instance of fragment MainFragment.
+     */
+    public static MainFragment newInstance()
```

Comment:
`newInstance` is a function that returns a newly created `MainFragment` object. What do you mean is it necessary to name this mainfragment?